correios_lib/base.py

```diff
@@ -58,6 +61,34 @@ def __init__(self, env, id_correios, password, cert=False):
         else:
             verify = certifi.where()
 
+        if log_config is not None and isinstance(dict):
```

Comment:
Aren't you missing the variable? 
`isinstance(log_config , dict)`
is this necessary?